lambdas/service/app.py

```diff
@@ -1538,22 +1562,27 @@ def handle_manifest_generation_request() -> Tuple[int, Manifest]:
                                                            catalog=catalog,
                                                            filters=filters)
         if manifest is not None:
-            return 0, manifest
+            return amend_manifest_location(manifest=manifest)
```

Comment:
Is there a reason to use the keyword arg here?
```suggestion
            return 0, amend_manifest_location(manifest)
```